docs/servers.rst

```diff
@@ -249,64 +215,59 @@ admin of the system (see `this GitHub
-Follow the recommendations as to choosing a strong passphrase. As the
-admin, you will be responsible for keeping this passphrase safe.
-Write it down somewhere and memorize it if you can. **If inadvertently
-lost it could result in total loss of the SecureDrop system.**
-
-After selecting either of those options you may be asked a few questions
-about overwriting anything currently on the server you are using. Select
-yes. You do not need an HTTP proxy, so when asked, you can just click
-continue.
-
-Finish the Installation
-~~~~~~~~~~~~~~~~~~~~~~~
+will occur after every nightly software update.
+
+Setting up storage
+~~~~~~~~~~~~~~~~~~
 
-Wait for the base system to finish installing. When you get to the
-*Configure tasksel* screen, choose **No automatic updates**. The
-subsequent SecureDrop installation will include a task that handles
-regular software updates.
+On the **Guided storage configuration** screen, verify that **Use an entire disk**
+is checked, and that the server's local disk is selected. Also verify that **Set
+up this disk as an LVM group** is selected.
 
-.. note:: The Ansible playbooks for SecureDrop will configure automatic
-          updates via ``cron-apt``. As part of the automatic update process,
-          the servers will reboot nightly. See the
-          :ref:`OSSEC guide <AnalyzingAlerts>` for example notifications
-          generated by the reboots.
+If you decided to set up FDE, despite the implications for administration overhead,
+select **Encrypt the LVM group with LUKS**, and enter and confirm the disk passphrase.
+Store this passphrase securely, as it will be required to unlock storage on every reboot.
 
-When you get to the software selection screen, deselect the preselected
-**Standard system utilities** and select **OpenSSH server** by highlighting each
-option and pressing the space bar.
+Select **Done** and press **Enter** to move to the **Storage Configuration** screen.
+Review the configuration and select **Done** and press **Enter** to continue. Then,
+choose **Continue** on the **Confirm destructive action** dialog.
 
-.. caution:: Hitting enter before the space bar will force you to start the
-             installation process over.
 
-Once **OpenSSH Server** is selected, hit *Continue*.
+Configure account and hostname
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the **Profile setup** screen, configure the server's hostname and the administration account:
 
-You will then have to wait for the packages to finish installing.
+- **Your name:** Specify the administrator account name, e.g. ``SecureDrop Admin``
+- **Your server's name:** Use ``app`` for the *Application Server*, and ``mon`` for
+  the *Monitor Server*
+- **Pick a username:** Specify the administrator account username, e.g. ``sdadmin``
+- **Choose a password:** Specify a strong password for the administrator account.
+  A Diceware-generated passphrase is recommended.
+- **Confirm your password:** Enter the password chosen above.
 
```

Comment:
Worth noting that the passwords for app and mon must be the same?

Reply:
Yup, fixed